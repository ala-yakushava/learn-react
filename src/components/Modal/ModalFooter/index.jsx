import React from 'react';
import PropTypes from 'prop-types';

import './style.scss';

const ModalFooter = ({ children }) => (
  <div className="ModalFooter">
    { children }
  </div>
);

ModalFooter.propTypes = {
  children: PropTypes.node.isRequired,
};

export default ModalFooter;
