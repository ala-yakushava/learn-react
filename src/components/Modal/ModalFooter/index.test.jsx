import React from 'react';
import renderer from 'react-test-renderer';

import ModalFooter from './index';

describe('Components - ModalFooter', () => {
  test('should match snapshot', () => {
    const tree = renderer
      .create(
        <ModalFooter>
          <div>Content</div>
        </ModalFooter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
