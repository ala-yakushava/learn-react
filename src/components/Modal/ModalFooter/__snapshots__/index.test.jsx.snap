// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - ModalFooter should match snapshot 1`] = `
<div
  className="ModalFooter"
>
  <div>
    Content
  </div>
</div>
`;
